public/app/plugins/panel/geomap/editor/DataLayersEditor.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { FC } from 'react';
+import { StandardEditorProps, MapLayerConfig } from '@grafana/data';
+import { GeomapPanelOptions } from '../types';
+import { LayerEditor } from './LayerEditor';
+
+// For now this supports a *single* data layer -- eventually we should support more than one
+export const DataLayersEditor: FC<StandardEditorProps<MapLayerConfig[], any, GeomapPanelOptions>> = ({
+  value,
+  onChange,
+  context,
+}) => {
+  return (
+    <LayerEditor
+      config={value?.length ? value[0] : undefined}
+      data={context.data}
+      onChange={(cfg) => {
+        console.log('Change overlays:', cfg);
+        onChange([cfg]);
```

Comment:
see that data layers is an array, thinking being able to stack multiple? (one coloring states, and another showing dots in specific locations for example)

Reply:
exactly.   UI only supports one right now, but multiple layers is very useful.

We will likely also want a common way to filter which frames (refId?) go to each layer